packages/entities/entities-plugins/src/components/PluginForm.vue

```diff
@@ -258,6 +269,7 @@ const treatAsCredential = computed((): boolean => !!(props.credential && props.c
 const record = ref<Record<string, any> | null>(null)
 const configResponse = ref<Record<string, any>>({})
 const formLoading = ref(false)
+const actionsDivRef = ref<HTMLDivElement>()
```

Comment:
```suggestion
const actionsDivRef = ref<HTMLDivElement | null>(null)
```

[Vue docs](https://vuejs.org/guide/typescript/composition-api.html#typing-component-template-refs) advise initializing template refs with `null`